python/cudf/cudf/core/frame.py

```diff
@@ -219,6 +220,41 @@ def _unaryop(self, op):
             result._data[name] = col.unary_operator(op)
         return result
 
+    def searchsorted(
+        self, values, side="left", ascending=True, na_position="last"
+    ):
+        """Find indices where elements should be inserted to maintain order
+
+        Parameters
+        ----------
+        value : Frame or Column (Shape must be consistent with self)
+            Values to be hypothetically inserted into Self
+        side : str {‘left’, ‘right’} optional, default ‘left‘
+            If ‘left’, the index of the first suitable location found is given
+            If ‘right’, return the last such index
+        ascending : bool optional, default True
+            Sorted Frame is in ascending order (otherwise descending)
+        na_position : str {‘last’, ‘first’} optional, default ‘last‘
+            Position of null values in sorted order
+
+        Returns
+        -------
+        1-D cupy array of insertion points
+        """
+        # Convert column-typed `values` to Series
+        if isinstance(values, ColumnBase):
+            from cudf.core.series import Series
+
+            values = Series(values)
```

Comment:
We shouldn't need to handle a Column of values in this function. If Column is being passed into here it should call `.as_frame()` first